enhancements/sig-policy/99-policy-placement-strategy/README.md

```diff
@@ -60,424 +55,481 @@ recommended to review it before proceeding.
+and waits for the cluster to rollout before resuming progress at the point previous to the new cluster being added.
+
+##### A New Cluster Is Added After a Rollout
+
+When a managed cluster is added to the placement after the rollout completed, the rollout does not restart. The cluster
+directly receives the last successful policy definition and has a rollout status of `NotApplicable`.
+
+##### A Cluster In The Group Is Offline
+
+If a cluster does not provide any compliance after the `progressDeadline` value has elapsed, the cluster is set to
+`TimeOut` and the rollout fails. Assuming no additional rollout is started, when the cluster comes back online, it will
+receieve the last successful policy version, if any.
+
+##### Retrying a Failed Rollout
+
+This is only necessary if `progressDeadline` is used since this is the only way a rollout can fail. Otherwise, the
+rollout indefinitely waits for the clusters to become compliant.
+
+In this situation, the first option is to update the policy spec in some way which would completly restart the rollout.
+
+The second option is to use the `spec.retryRollout.rolloutUID` field on the `Rollout` object. When set to a UID that
+matches `status.rolloutUID`, the Governance Policy Propagator will restart the rollout and create a new rollout UID.
+This is used instead of a boolean that gets automatically flipped back to `false` to support a GitOps workflow.
+
+```yaml
+apiVersion: v1beta1
+kind: Rollout
+metadata:
+  name: policy-<same as root policy>
+  namespace: <same as root policy>
+spec:
+  retryRollout:
+    rolloutUID: <UID from status.rolloutUID>
+```
 
-#### Story 2
+To support this, the `Rollout` will have UID in its status that gets generated on every new rollout.
 
-As a system administrator, I want to specify a placement `DecisionStrategy` bound with policies, configure the order of
-the updates, and continue enforcement rollout only when the policy on previous clusters show as "Compliant".
+```yaml
+apiVersion: v1beta1
+kind: Rollout
+metadata:
+  name: policy-<same as root policy>
+  namespace: <same as root policy>
+status:
+  rolloutUID: 3ac24863-c829-4a0d-9ea8-050c9c4debf6
+```
 
-- **Summary**
+##### Ignoring Clusters During a Rollout
+
+If there are clusters that are acting up and shouldn't impact the rollout from proceeding, the user can set a label
+selector on the `spec.rolloutStrategy.ignoreClusterRolloutStatus` field on the `Policy` to match such clusters. They
+will still receive the updated policy but their rollout status does not prevent the rollout from proceeding. If
+`progressDeadline` is not set, then the rollout does not wait for a rollout status from this cluster.
+
+```yaml
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: sample-policy
+  namespace: policy-namespace
+spec:
+  disabled: false
+  remediationAction: enforce
+  rolloutStrategy:
+    ignoreClusterRolloutStatus:
+      matchExpressions:
```

Comment:
Why use this instead of simple group name?

Reply:
You might have prod clusters that are flaky but you still want to apply the policy to them without ignoring the status of all prod clusters.